src/Compilers/CSharp/Portable/BoundTree/IBoundInvalidNode.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal interface IBoundInvalidNode
```

Comment:
>IBoundInvalidNode [](start = 23, length = 17)

It would be good to add a doc comment explaining the purpose and when it is appropriate to implement this interface.